tests/manual/performance/setdata.html

```diff
@@ -0,0 +1,11 @@
+<div id="test-controls">
```

Comment:
Those files are mangled from another PR. For atomic PRs we can't create branches from one another (only from `master`). The problem might be shared code but for those quick PRs we should minimize the round-trip time for review.

ps.: I'll see how this will look after merging https://github.com/ckeditor/ckeditor5/pull/6103.